ground-server/src/app/grpc/commandClient.js

```diff
@@ -1,41 +1,43 @@
 "use server";
+
 var PROTO_PATH = __dirname + '/../../../../../protos/command.proto';
 var grpc = require('@grpc/grpc-js');
```

Comment:
Change to 

import * as grpc from '@grpc/grpc-js';
import * as protoLoader from '@grpc/proto-loader';

for ESLint